File: templates/backend/fastapi/app/routes/app_info.py
from fastapi import APIRouter, HTTPException

from app.db import get_app_info

router = APIRouter()


@router.get("")
async def read_app_info():
    app_info = await get_app_info()
    if not app_info:
        raise HTTPException(status_code=404, detail="AppInfo has not been seeded yet.")
    return app_info

File: templates/backend/express/src/routes/app-info.js
import { Router } from "express";

import { getAppInfo } from "../db.js";

const router = Router();

router.get("/", async (_req, res) => {
  try {
    const appInfo = await getAppInfo();
    if (!appInfo) {
      res.status(404).json({ error: "AppInfo has not been seeded yet." });
      return;
    }
    res.json(appInfo);
  } catch (error) {
    res.status(500).json({ error: "Failed to load app info.", message: error.message });
  }
});

export default router;
